### DISCUSSION TITLE
Unable to use tts_models--cs--cv--vits to do end-to-end synthesizing

### DISCUSSION THREAD

**Discussion by yaoyuan0801** [coqui-ai/TTS#2417] [General Q&A]
The code is 
```
tts.tts_to_file("这是真的吗？我有点不敢相信呢。", speaker_wav="~/voice_clone/sample.wav", file_path="output.wav")
```
And the failure due to `encoder_model_path` and `encoder_config_path` are both empty string. This means encoder could not be initialized.

Any idea?

**Comment by erogol** [comment]
cs is a Czech model FYI
